spec/services/arcgis_api/geocoder_spec.rb

```diff
@@ -156,5 +156,14 @@
       expect(WebMock).to have_requested(:get, %r{/suggest}).
         with(headers: { 'Authorization' => 'Bearer token1' }).twice
     end
+
+    it 'manually sets the expiration if the cache store is redis' do
+      stub_generate_token_response
+      redis = double('Redis cache store')
+      expect(redis).to receive(:expire).once
+      expect(Rails.cache).to receive(:redis).and_return(redis)
+
+      subject.retrieve_token!
+    end
```

Comment:
what if we configed `Rails.cache` to use a redis instance and then checked the `ttl` in the backing redis?
```suggestion
    context 'when using redis as a backing store' do
      around do |ex|
        original = Rails.cache.store
        Rails.cache.store = ActiveSupport::Cache::RedisCacheStore.new(
          url: IdentityConfig.store.redis_throttle_url,
        )
        ex.run
      ensure
        Rails.cache.store = original
      end
      
      it 'manually sets the expiration if the cache store is redis' do
        stub_generate_token_response
        subject.retrieve_token!
        ttl = Rails.cache.redis.ttl(ArcGisApi::Geocoder:: API_TOKEN_CACHE_KEY)
        expect(ttl).to be > 0
    end
```

Reply:
My guesses in the suggestion were off, so I downloaded the branch, and got it working and pushed 88ec1922162054dc3d7c0becbbcfa8e94226bb43